frontend/components/cards/ModuleUnionCard.tsx

```diff
@@ -8,6 +9,25 @@ interface ModuleUnionCardProps {
   selected?: boolean;
 }
 
+interface HighlightEllipsisProps {
+  unsafeHTML: string;
+}
+
+const HighlightEllipsis: FC<HighlightEllipsisProps> = ({ unsafeHTML }) => (
+  <Box
+    sx={{
```

Comment:
I've noticed that every once in a while, we still get unexpected HTML elements with block display (e.g., `p` tags). Here's an example:
![Screen Shot 2021-09-11 at 12 17 44](https://user-images.githubusercontent.com/25336795/132956131-8bd8bcb3-44ca-47b9-8087-e6fea39708e4.png)

I think for the content snippets, it would be best if we displayed the content inline (i.e., with `display: inline`, but included ellipses so that the content isn't jammed together incomprehensibly. Something like this:
![Screen Shot 2021-09-11 at 12 30 13](https://user-images.githubusercontent.com/25336795/132956230-19e20198-2dc7-41da-a244-99fc1e664325.png)

Reply:
Not sure if that fixed the problem as I was still seeing some leading ellipsis in content highlights, but I may fix that in the next sprint in a short task. Your fix has been implemented though.